### PR TITLE
Turn exercise 024 to practice

### PR DESCRIPTION
**Pull request by Ozyugoo**

### CODE REVIEW

**Comment by Ozyugoo** [comment]
Hi, @cuihtlauac kindly check it out.

File: practice/024/answer/impl.ml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+let range a b =
+  let rec aux acc high low =
+    if high >= low then
+      aux (high :: acc) (high - 1) low
+    else acc
+  in
+    if a < b then aux [] b a else List.rev (aux [] a b)
+
+  let lotto_select n m = rand_select (range 1 m) n
```
Comment (cuihtlauac) [inline practice/024/answer/impl.ml]:
```suggestion
  let lotto_select n m = rand_select (range 1 m) n

```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: practice/024/answer/impl.ml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
```
Comment (cuihtlauac) [inline practice/024/answer/impl.ml]: You need to fix this end of file

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: practice/024/answer/impl.ml (hunk 2 above)
Comment (Ozyugoo) [reply practice/024/answer/impl.ml]: Hey @cuihtlauac sorry about this end of file, I will be more careful. I was viewing with the wrong editor as you said.

**Review by cuihtlauac — APPROVED** [review]
Thanks @Ozyugoo, this looks good to me

**Pull request merged**